### ISSUE TITLE
Rocksdb write duration is increasing with data growth

### ISSUE THREAD

**Issue by Connor1996** [tikv/tikv#8722]
![image](https://user-images.githubusercontent.com/13497871/93966939-91ba8d80-fd98-11ea-9881-eb808f9d737b.png)
kvdb's max write duration is increasing (note that compaction filter is enabled)

![image](https://user-images.githubusercontent.com/13497871/93967055-e78f3580-fd98-11ea-80c5-8ef97dc8162c.png)
from perf context, it shows that pre_and_post_process contibutes most of increasing duration

**Comment by nolouch** [comment]
Is there any way we can solve this problem?

**Comment by Connor1996** [comment]
@nolouch No much idea on this now. @hicqu Do you have observed this issue when testing the compaction filter?

**Comment by yiwu-arbug** [comment]
Does write duration increase during the period? Could it be writers waiting in queue, waiting for previous writes to finish? In that case, maybe tuning write group batch size could help.

**Comment by Connor1996** [comment]
> Does write duration increase during the period? Could it be writers waiting in queue, waiting for previous writes to finish? In that case, maybe tuning write group batch size could help.

Yes, pre_and_post_process of write duration is increasing. You can see it in the above images.

**Comment by yiwu-arbug** [comment]
what use case is this? is `force_consistency_checks` disabled? If not we need to disable it.
